common/datavalues/src/types/data_type.rs

```diff
@@ -80,6 +100,11 @@ impl DataType {
                 ArrowDataType::Struct(arrows_fields)
             }
             String => ArrowDataType::LargeBinary,
+            Interval(unit) => ArrowDataType::Extension(
```

Comment:
let's just use `ArrowDataType::Int64`, extension logical type currently is not allowed in parquet write.

Reply:
We can add that as an issue in arrow2: we work on that there.

Reply:
Thanks, already addressed in https://github.com/jorgecarleitao/arrow2/issues/392